### DISCUSSION TITLE
.txt

### DISCUSSION THREAD

**Discussion by lkw052715** [obss/sahi#946] [Q&A]
How do I get the inferred label file?

**Comment by andressrodrl** [comment]
You cant, you can only save results (in coco format) if you provide the coco labels. However I have a repo where i implemented the feature of saving results in yolo format (with or without confidence).

https://github.com/andresinsitu/sahi_custom/tree/export_txt

You can save the results in yolo format as follows:

Add the argument `export_yolo=True` or the argument `export_yolo_conf=True` to the predict function. The first one only saves xywh and the second one also saves confidence so xywhc . Results will be saved to a "labels" folder
